sentences/de/fan_HassTurnOff.yaml

```
@@ -3,19 +3,23 @@ intents:
   HassTurnOff:
     data:
       - sentences:
-          - "schalte <luefter> <area> <aus>"
-          - "schalte <area> <luefter> <aus>"
+          - "<schalten> <luefter> <area> <aus>"
+          - "<schalten> <area> <luefter> <aus>"
           - "stoppe <luefter> <area> [<aus>]"
           - "stoppe <area> <luefter> [<aus>]"
-          - "(mach|mache) <luefter> <area> <aus>"
-          - "(mach|mache) <area> <luefter> <aus>"
+          - "<machen> <luefter> <area> <aus>"
+          - "<machen> <area> <luefter> <aus>"
+          - "<luefter> <area> <aus>[<schalten>|<machen>]"
+          - "<area> <luefter> <aus>[<schalten>|<machen>]"
 
-          - "schalte <alle_luefter> [<area>] <aus>"
-          - "schalte [<area>] <alle_luefter> <aus>"
+          - "<schalten> <alle_luefter> [<area>] <aus>"
+          - "<schalten> [<area>] <alle_luefter> <aus>"
```

```suggestion
          - "<schalten> <area> <alle_luefter> <aus>"
```

i know, it was already before, but it does not make sense to make `<area>` optional here, because this is already covered above